### ISSUE TITLE
Improve messaging of what Fair Source is

### ISSUE THREAD

**Issue by dcramer** [fairsource/fair.io#35]
There's a couple points we really need to nail:

1. Open Source advocates should not be anti Fair Source, so remove the ability for them to be. Nothing we're doing is anti OSS, its pro OSS and pro free access to software. Look at all languages, all complaints, find ways to remove them.

2. Make it way more obvious what Fair Source is on the homepage, less corporate-y. For example, lessen this focus on "engage the developer community" (what are we engaging? hows it diff from closed source) and put more emphasis on maximizing user access, while protecting maintainer's. Ideally its way more obvious what we're about so that there's less "i didnt read anything" arguments.

One of the most frequent lines of wrongly-complaints I see right now is that Fair Source is trying to persuade people to not be Open Source, but thats simply not the case. Rather we're trying to create more access to free software, more open source software, specifically for the people who are afraid to Open Source their business.

**Comment by ezekg** [comment]
I also think we need an FAQ on "How does Fair Source relate to source-available?" because it's a common question, at least coming from somebody vested in Open Source (i.e. from those that even know the term "source-available" exists).

Also, the "Why should I adopt Fair Source at my company?" FAQ needs to be cleaned up. It's very corporate-speak.

**Comment by ezekg** [comment]
Also, I think "About" should be changed to "Definition", or another link to the definition should be added. In my discussions, it's been a common thing for people to entirely miss the definition, causing confusion or misunderstandings.

**Comment by chadwhitacre** [comment]
Let's bring [this feedback](https://x.com/ForrestTheWoods/status/1821998478102556999) in here as well:

> I’ve clicked all through the Fair Source website and for the life of me I can’t figure out exactly what is or isn’t allowed. Wish the descriptions were much more explicit.
>
>> allows use, modification, and redistribution with minimal restrictions
>
> what are the restrictions!? I think what I really need are examples of what is or isn't allowed. And a nice table comparing the different "Fair Core" licenses.
> 
> I had put FSL and FCL in a text diff tool to figure out how they were different. Very annoying to have to do that.

Diff addressed in #40. We had a chart but dropped it when we simplified down to three licenses. I think that's still okay because BUSL is basically all over the place so not neat and tidy to compare anyway.

For examples of what each licenses allows there's a question of how much we want to explain on fair.io vs. handling on the respective license websites.

**Comment by ezekg** [comment]
Do you think a chart of open source, fair source, source-available, and closed source would be valuable in communicating where fair source lands in terms of "openness"? There's some obvious growing pains here, but some visualization may help.

```
     |------------------------------------------------------------------|
     ^         ^                        ^                               ^
     |         |                        |                               |
     |    fair source           source-available                        |
open source                                                       closed source
```

But then again that ~~may~~ will be construed as openwashing. 🙃

**Comment by ezekg** [comment]
Yeah, I think DOSP is really what brings fair source so close to open source in terms of openness.

**Comment by chadwhitacre** [comment]
First PR in #45. 👀

**Comment by Dieterbe** [comment]
> I think such a graph would be interesting, but I think it does not fully address the DOSP element of it. I'm not sure how you best visualize the sliding window part of it best.

my attempt is here: https://dieter.plaetinck.be/files/fair-source-open-core-and-hybrids.png
![fair-source-open-core-and-hybrids](https://github.com/user-attachments/assets/d3e61707-2c92-4cda-9248-b69f2a0a6a8e)

(this was part of my [blog post](https://dieter.plaetinck.be/posts/fair-source-the-next-best-model-for-commercial-open-source/) where i compare fair source to open core, which has some similarities with fair source)

**Comment by jstumpin** [comment]
> Do you think a chart of open source, fair source, source-available, and closed source would be valuable in communicating where fair source lands in terms of "openness"? There's some obvious growing pains here, but some visualization may help.
> 
> ```
>      |------------------------------------------------------------------|
>      ^         ^                        ^                               ^
>      |         |                        |                               |
>      |    fair source           source-available                        |
> open source                                                       closed source
> ```
> 
> But then again that ~may~ will be construed as openwashing. 🙃

`list 5 top shared features amongst fair source, open source, and closed source software licenses. associate each feature with a value of 0 to 5, 5 being highest score. draw a radar chart for each software license on top of each other based on the scores that you assigned. please ensure the graph contains all three software licenses.`

![blob](https://github.com/user-attachments/assets/d203c9b5-b916-467f-94ae-ef0200daf2b1)

ChatGPT numerously failed to generate a proper chart when there are four licenses, hence source-available omission 😅

**Comment by manchicken** [comment]
> 1. Open Source advocates should not be anti Fair Source, so remove the ability for them to be. Nothing we're doing is anti OSS, its pro OSS and pro free access to software. Look at all language, all complaints, find ways to remove the ones that are at least grounded in miscommunication.

It does seem inappropriate to tell other people how to feel about your novel approach to Shareware. A software license that behaves differently in different contexts, or discriminates against different users, is indeed hostile to the Open Source ethos as well as the Four Freedoms.
